### PR TITLE
update CI test manifests for OV GPU plugin

### PR DESCRIPTION
**Pull request by bani-intelaipg**
* Update GPU manifests to be independent from CPU manifests
* In build_tf.py make intel-mkl-tf as NOT default

### CODE REVIEW

File: build_tf.py [hunk 1]
```diff
@@ -71,7 +71,7 @@ def main():
         os.chdir(pwd)
 
     use_intel_tf = False
-    if arguments.use_intel_tensorflow != '':
+    if arguments.use_intel_tensorflow:
         use_intel_tf = True
```
Comment (adk9) [inline build_tf.py:75]: This doesn't change the default. Please change the default to false.

File: examples/cpp/infer_single_network.cc [hunk 2]
```diff
@@ -59,15 +59,6 @@ void PrintAvailableBackends() {
...
 
-// Sets the specified backend. This backend must be set BEFORE running
-// the computation
-tf::Status SetNGraphBackend(const string& backend_name) {
-  // Select a backend
-  tf::Status status =
-      tf::ngraph_bridge::BackendManager::SetBackend(backend_name);
-  return status;
-}
-
```
Comment (adk9) [inline examples/cpp/infer_single_network.cc:70]: Why do you have these changes again?

**Review by adk9 — CHANGES_REQUESTED** [review]

File: build_tf.py (hunk 1 above)
Comment (bani-intelaipg) [reply build_tf.py:75]: The action="strore_true" will set it as True only if the arg is specified

File: examples/cpp/infer_single_network.cc (hunk 2 above)
Comment (bani-intelaipg) [reply examples/cpp/infer_single_network.cc]: merge issues, resolved.

**Comment by bani-intelaipg** [comment]
@googlebot I consent

**Comment by adk9** [comment]
@googlebot I consent

**Review by adk9 — APPROVED** [review]
LGTM

**Pull request merged**